services/areas.js

```diff
@@ -67,7 +68,9 @@ export const getAreas = () =>
 export const saveArea = (data) =>
   apiAuthRequest({
     method: data.id ? 'PATCH' : 'POST',
-    url: data.id ? `${REQUEST_URL}/${data.id}` : REQUEST_URL,
+    url: data.id
+      ? `${REQUEST_URL}/${data.id}${SOURCE}`
+      : `${REQUEST_URL}${SOURCE}`,
```

Comment:
These are `PATCH` and `POST` URLs. They don't need to have the new query params.
For these, a change in the body will be needed but, it's not part of the scope of this card.

Reply:
I just realized I replicated the source for all methods instead of GET only haha thank you for notice it!